helm/envoy-gateway/Chart.yaml

```diff
@@ -1,20 +1,17 @@
 apiVersion: v2
-appVersion: 1.0.2
-name: envoy-gateway
-
-### ....
-description: Envoy Gateway
-
+appVersion: v1.0.2
+description: The Helm chart for Envoy Gateway
 home: https://github.com/giantswarm/envoy-gateway-app
-
-# If you have an icon/logo, you should add it to https://github.com/giantswarm/web-assets
-# and set the final URL as a value here and uncomment.
-icon: https://s.giantswarm.io/app-icons/example/1/light.svg
-
+icon: https://s.giantswarm.io/app-icons/giantswarm/1/light.svg
+keywords:
+- gateway-api
+- envoyproxy
+- envoy-gateway
+- eg
+annotations:
+  application.giantswarm.io/team: cabbage
+name: envoy-gateway
 sources:
 - https://github.com/envoyproxy/gateway
-
+type: application
 version: 0.0.0-dev
```

Comment:
I don't get why you're setting 0.0.0-dev here? In context to https://github.com/giantswarm/cilium-app/pull/195#discussion_r1686140477

Reply:
because there is no release yet for this app. it's part of the template